crates/bevy_pbr/src/entity.rs

```diff
@@ -40,10 +44,18 @@ impl Default for PbrBundle {
     }
 }
 
-/// A component bundle for "light" entities
+/// A component bundle for "point light" entities
 #[derive(Debug, Bundle, Default)]
 pub struct PointLightBundle {
     pub point_light: PointLight,
     pub transform: Transform,
     pub global_transform: GlobalTransform,
 }
+
+/// A component bundle for "directional light" entities
+#[derive(Debug, Bundle, Default)]
+pub struct DirectionalLightBundle {
+    pub directional_light: DirectionalLight,
+    pub transform: Transform,
+    pub global_transform: GlobalTransform,
```

Comment:
Just as a note this may have to get updated if #2331 lands :)
So we can utilize `TransformBundle`